gemini/sample-apps/fashion-trends-prediction/app/data_prep/create_new_data.py

```diff
@@ -0,0 +1,59 @@
+import json
+import os
+
+from config import config
+from dataProcessing import generateCaption
+from driver import get_top_categories
+
+data_path = config["Data"]["current_data"]
+
+
+def imagesScrape(saved):
```

Comment:
Add typing for all function parameters and return types

Reply:
Done